tests/testthat/test_clone.R

```diff
@@ -210,99 +290,208 @@ test_that("Test hierarchicalClones only_heavy and first", {
+            summarize_clones = TRUE,
+            first = T, # default is first=F
+            nproc = 1
+        ),
+        "Running defineClonesScoper in single cell mode",
+        fixed = TRUE
+    )
+    expect_equal(clones@db[["clone_id"]], c("1", "1", "1", "1", "1", "1", "2", "2"))
+
     # Case 3
-    expect_warning(clones <- hierarchicalClones(
-        db,
-        threshold=0,
-        cell_id='cell_id',
-        locus='locus',
-        only_heavy=FALSE,
-        split_light=TRUE,
-        summarize_clones=TRUE,
-        first=F, # default is first=F
-        nproc=1))
-    expect_true(all(clones@db[['clone_id']] == "1"))
+    expect_message(
+        expect_warning(
+            expect_warning(
+                clones <- hierarchicalClones(
+                    db,
+                    threshold = 0,
+                    cell_id = "cell_id",
+                    locus = "locus",
+                    only_heavy = FALSE,
+                    split_light = TRUE,
+                    summarize_clones = TRUE,
+                    first = F, # default is first=F
+                    nproc = 1
+                ),
+                "split_light = TRUE is deprecated. Please use split_light = FALSE. After clonal identification, light chain groups can be found with dowser::resolveLightChains"
+            ),
+            "only_heavy = FALSE is deprecated. Running as if only_heavy = TRUE"
+        ),
+        "Running defineClonesScoper in single cell mode",
+    )
+
+    expect_true(all(clones@db[["clone_id"]] == "1"))
     
     # Case 4
-    expect_warning(clones <- hierarchicalClones(
-      db,
-      threshold=0,
-      cell_id='cell_id',
-      locus='locus',
-      only_heavy=FALSE,
-      split_light=TRUE,
-      summarize_clones=TRUE,
-      first=T, # default is first=F
-      nproc=1))
+    expect_warning(
+        expect_warning(
+            expect_message(
+                clones <- hierarchicalClones(
+                    db,
+                    threshold = 0,
+                    cell_id = "cell_id",
+                    locus = "locus",
+                    only_heavy = FALSE,
+                    split_light = TRUE,
+                    summarize_clones = TRUE,
+                    first = T, # default is first=F
+                    nproc = 1
+                ),
+                "Running defineClonesScoper in single cell mode",
+                fixed = TRUE
+            ),
+            "only_heavy = FALSE is deprecated. Running as if only_heavy = TRUE"
+        ),
+        "split_light = TRUE is deprecated. Please use split_light = FALSE. After clonal identification, light chain groups can be found with dowser::resolveLightChains"
+    )
+
     expect_equal(clones@db[['clone_id']],c("1","1","1","1","1","1","2","2"))
     
-
-    #TODO: consider merging the dataset here with the one from the test above to avoid duplication.
-    #TODO: check if comment is addressed.
+    #TODO: check in dowser if comment is addressed.
```

Comment:
Hi @colejensen! Please, review this PR and the split_light tests and comments (like the one below this comment). As split_light is not supported now, some of the questions need to be tested after using dowser::resolveLightChains. If the questions are addressed in dowser, we can remove the comments/tests.